actk/steps/standardize_fov_array/standardize_fov_array.py

```diff
@@ -181,6 +181,24 @@ def run(
         # Log original length of cell dataset
         log.info(f"Original dataset length: {len(dataset)}")
 
+        # Check assumption: all fields per FOV are constant
+        # except CellID and CellIndex
+        const_cols_per_fov = [
+            c for c in dataset.columns if c not in ["CellId", "CellIndex"]
+        ]
+        df_const_cols = dataset.groupby("FOVId")[const_cols_per_fov].nunique().eq(1)
+
+        for col_name, is_const in df_const_cols.all().iteritems():
+            try:
+                assert is_const
+            except AssertionError:
+                example = df_const_cols[~df_const_cols[col_name]].sample()
+                logging.error(
+                    f"{col_name} has multiple values per FOV."
+                    f"Example: FOV {example.index.item()}",
+                    exc_info=True,
+                )
```

Comment:
This won't continue to raise the exception will it? Do you want to log the exception and continue or fully raise the exception?

Reply:
i'd like the failed assertion to halt the step -- do i need to `raise` to do that?

Reply:
I would `raise ValueError({your current message})`

Reply:
i think that works but now my test fails because there's an error?  how should i catch it in the test?